pkg/api/api.go

```diff
@@ -279,9 +279,20 @@ func (a *API) RegisterChunksPurger(store *purger.DeleteStore, deleteRequestCance
 	a.RegisterRoute(path.Join(a.cfg.LegacyHTTPPrefix, "/api/v1/admin/tsdb/cancel_delete_request"), http.HandlerFunc(deleteRequestHandler.CancelDeleteRequestHandler), true, "PUT", "POST")
 }
 
-func (a *API) RegisterTenantDeletion(api *purger.TenantDeletionAPI) {
-	a.RegisterRoute("/purger/delete_tenant", http.HandlerFunc(api.DeleteTenant), true, "POST")
-	a.RegisterRoute("/purger/delete_tenant_status", http.HandlerFunc(api.DeleteTenantStatus), true, "GET")
+func (a *API) RegisterBlocksPurgerAPI(blocksPurger *purger.BlocksPurgerAPI) {
+
+	a.RegisterRoute(path.Join(a.cfg.PrometheusHTTPPrefix, "/api/v1/admin/tsdb/delete_series"), http.HandlerFunc(blocksPurger.V2AddDeleteRequestHandler), true, "PUT", "POST")
```

Comment:
These functions exist for the chunks purger. I thought adding the V2 like in the ingester functions would make things more clear. They are in the `blocksPurger` so that might be enough of information to tell them apart. I will remove it.